build/pipelines/templates/run-ui-tests.yaml

```diff
@@ -39,8 +35,10 @@ jobs:
       filePath: $(Build.ArtifactStagingDirectory)/drop/Release/${{ parameters.platform }}/Calculator/AppPackages/Calculator_$(Build.BuildNumber)_Test/Add-AppDevPackage.ps1
       arguments: -Force
 
-  - powershell: Start-Process -FilePath "C:\Program Files (x86)\Windows Application Driver\WinAppDriver.exe" -Verb RunAs
-    displayName: Start WinAppDriver
+  - task: WinAppDriver.winappdriver-pipelines-task.winappdriver-pipelines-task.Windows Application Driver@0
+    displayName: 'Start - WinAppDriver'
```

Comment:
From the perspective of later perusing the pipeline state, it might make a bit more sense to see `WinAppDriver - Start` and `WinAppDriver - Stop`.  They'll associate to each other a little easier.

Reply:
Gotcha - that makes sense. I've updated the displayName for both Start and Stop accordingly. Thanks!

Reply:
I actually liked it better the way you originally did it :) but I think either way is fine, let's just leave it like this.